config.json

```diff
@@ -675,12 +675,14 @@
         "slug": "accumulate",
         "name": "Accumulate",
         "uuid": "fb03948c-11e8-440d-9a5d-979396451270",
+        "concepts": [],
```

Comment:
The concepts key only exists for concept exercises. It does not need to be added here.

Reply:
Oh okay